README.md

```diff
@@ -26,8 +26,6 @@ finally makes using a blockchain protocol easy for both developers and consumers
 
 [Check out quick start documentation](https://docs.nearprotocol.com/quick-start), specifically:
   - [Working with Smart Contracts in NEAR Studio](https://docs.nearprotocol.com/working-smart-contracts)
-  - [Running local DevNet](https://docs.nearprotocol.com/development-environment/advanced)
-  - [Join TestNet](https://docs.nearprotocol.com/development-environment/run-your-own-testnet)
```

Comment:
Since nearcore is completely nightshade, it probably makes sense to link to the nightshade versions of these links instead of entirely blowing them away.

Reply:
That's handled in the `running` section of the readme where we link to how to run remotely and run locally

Reply:
https://github.com/nearprotocol/nearcore#running